qa/rpc-tests/getchaintips.py

```diff
@@ -9,7 +9,7 @@
 # This gives us two tips, verify that it works.
 
 from test_framework.test_framework import BitcoinTestFramework
-from test_framework.util import assert_equal
```

Comment:
nit: why import everything?

Reply:
because i was feeling momentarily lazy...i'll modify it and just use "time"